### ISSUE TITLE
Execution of Task211020CreateHostIntegrityCheckerResultTables Fails on MSSQL

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#22605]
**Describe the bug**
The execution of the upgrade task fails with the following message 

```
dotcms_fresh-dotcms-dev-1 | 20:54:30.474 INFO db.DotConnect - Executing CREATE TABLE hosts_ir( local_identifier VARCHAR(36), remote_identifier VARCHAR(36), endpoint_id VARCHAR(40), local_working_inode VARCHAR(36), local_live_inode VARCHAR(36), remote_working_inode VARCHAR(36), remote_live_inode VARCHAR(36), language_id int8, host VARCHAR(255), PRIMARY KEY (local_working_inode, language_id, endpoint_id))
dotcms_fresh-dotcms-dev-1 | 20:54:30.478 FATAL runonce.Task211020CreateHostIntegrityCheckerResultTables - Unable to execute SQL upgrade
dotcms_fresh-dotcms-dev-1 | com.microsoft.sqlserver.jdbc.SQLServerException: Column, parameter, or variable #8: 
Cannot find data type int8.
```

At first glance, it seems the UT is applying the Postgres script on mssql which is completely odd. But once you look closer you'll see the upgrade task has a method like: 

```
public String getMSSQLScript() {
        return CREATE_HOSTS_IR_TABLE_SCRIPT.replaceAll("INT8", "NUMERIC(19,0)");
}
```

That is meant to replace the postgres specif data-type int8 with NUMERIC(19,0)

The problem here is that the original script in code that looks like this

```
private static final String CREATE_HOSTS_IR_TABLE_SCRIPT = "CREATE TABLE hosts_ir(" +
            " local_identifier VARCHAR(36)," +
            " remote_identifier VARCHAR(36)," +
            " endpoint_id VARCHAR(40)," +
            " local_working_inode VARCHAR(36)," +
            " local_live_inode VARCHAR(36)," +
            " remote_working_inode VARCHAR(36)," +
            " remote_live_inode VARCHAR(36)," +
            " language_id int8," + 
            " host VARCHAR(255)," +
            " PRIMARY KEY (local_working_inode, language_id, endpoint_id)" +
            ");\n";
```

has the datatype **int8 in lower case.** Therefore the method that is meant to correct and replace the invalid data type is not accomplishing its job successfully. The method getMSSQLScript only takes care of UPPERCASE occurrences 

The problem was reported by @yolabingo he encountered the problem, migrating an environment from 5.2 to 22.03.

**To Reproduce**
1. Set up a dotCMS instance using MSSQL with 5.2 Let it settle and create it's db
2. Now set up the instance using 22.03 and keep the same db. The migration should fail 

**Expected behavior**
   The migration should succeed

**Comment by dsilvam** [comment]
Passed Internal QA: Tested upgrading from 21.11 to 22.07 via docker with MSSQL db.

**Comment by bryanboza** [comment]
Fixed, tested on release-22.07 // Docker // FF

**Comment by erickgonzalez** [comment]
UT doesn't exist in 5.3.8 or 21.06 LTS
